x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts

```diff
@@ -107,11 +109,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await pageObjects.embeddedConsole.clickEmbeddedConsoleControlBar();
       });
 
-      // FLAKY: https://github.com/elastic/kibana/issues/197144
-      describe.skip('With data', () => {
+      describe('With data', () => {
         before(async () => {
           await es.index({
             index: indexName,
+            refresh: true,
```

Comment:
not sure this is enough of a fix, we probably want to leave this skipped until we can so a larger refactor on these tests.

Reply:
As I've got the issue was flaky because when we update data this appear in kibana in random moment and makes test flaky. So I hope this one will permanently fix it

Reply:
Also we can try to load index with data in it
`await esArchiver.load(esArchiveIndex)`

Reply:
https://github.com/elastic/search-team/issues/8627#issuecomment-2476826752